sdk/spring/azure-spring-boot-starter/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 3.7.0-beta.1 (Unreleased)
+## 3.7.0 (2021-07-20)
+No changes from 3.6.1.
```

Comment:
Should this be align with azure-spring-boot that cover all changes of spring boot starters？

Reply:
changed.